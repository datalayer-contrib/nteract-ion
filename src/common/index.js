// @flow
export * from "./elevation";
export * from "./intent";
