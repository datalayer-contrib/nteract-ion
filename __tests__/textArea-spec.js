// @flow
import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import { TextArea, Intent } from "../src";

describe("TextArea", () => {
  it("renders a simple text area", () => {
    const component = shallow(
      <TextArea fill={true} large={4} intent={Intent.primary} />
    );
    expect(toJson(component)).toMatchSnapshot();
  });
});
