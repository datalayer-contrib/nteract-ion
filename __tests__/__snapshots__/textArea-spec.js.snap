// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea renders a simple text area 1`] = `
<ThemeProvider
  theme={
    Object {
      "shadows": Object {
        "danger": "0 0 0 0 rgba(219, 55, 55, 0), 0 0 0 0 rgba(219, 55, 55, 0), inset 0 0 0 1px #db3737, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2)",
        "high": "0 0 0 1px rgba(16, 22, 26, 0.1), 0 2px 4px rgba(16, 22, 26, 0.2), 0 8px 24px rgba(16, 22, 26, 0.2)",
        "highest": "0 0 0 1px rgba(16, 22, 26, 0.1), 0 4px 8px rgba(16, 22, 26, 0.2), 0 18px 46px 6px rgba(16, 22, 26, 0.2)",
        "low": "0 5px 20px rgba(0, 0, 0, 0.02), 0 0px 1px rgba(0, 0, 0, 0.9)",
        "middle": "0 0 0 1px rgba(16, 22, 26, 0.1), 0 1px 1px rgba(16, 22, 26, 0.2), 0 2px 6px rgba(16, 22, 26, 0.2)",
        "none": "",
        "plain": "0 0 0 .25px	rgb(128,128,128, .9)",
        "primary": "0 0 0 0 rgba(19, 124, 189, 0), 0 0 0 0 rgba(19, 124, 189, 0), inset 0 0 0 1px #137cbd, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2)",
        "success": "0 0 0 0 rgba(15, 153, 96, 0), 0 0 0 0 rgba(15, 153, 96, 0), inset 0 0 0 1px #0f9960, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2)",
        "warning": "0 0 0 0 rgba(217, 130, 43, 0), 0 0 0 0 rgba(217, 130, 43, 0), inset 0 0 0 1px #d9822b, inset 0 0 0 1px rgba(16, 22, 26, 0.15), inset 0 1px 1px rgba(16, 22, 26, 0.2)",
      },
    }
  }
>
  <styled.textarea
    border="none"
    borderRadius={3}
    boxShadow="primary"
    fontSize={4}
    width={1}
  />
</ThemeProvider>
`;
